p/themes/base-theme/template.css

```diff
@@ -93,10 +93,14 @@ img.favicon {
 }
 
 /*=== Videos */
-iframe, embed, object, video {
+audio, iframe, embed, object, video {
 	max-width: 100%;
 }
 
+audio {
+	width: 100%;
```

Comment:
Doesn't this override the max-width above? (can't check atm)

Reply:
`max-width` wins over `width`. I am not sure we actually need the `max-width` rule, but it was mainly to be consistent with the rules used for `video` etc.